docs/core/howto/python3.rst

```diff
@@ -4,7 +4,7 @@ Porting to Python 3
 Introduction
 ------------
 
-Twisted currently supports only Python 3.5+.
+Twisted currently supports only Python 3.6+.
```

Comment:
Not sure if we should keep this line... I guess that by this time, someone will already know the supported Python3 version for Twisted

Reply:
I'm going to err on the side of keeping it for now